client/status/v2_test.go

```diff
@@ -158,5 +222,17 @@ func TestFromStatusV2(t *testing.T) {
 			// must generate the same status.Status message
 			require.Equal(t, stv2, res)
 		}
+
+		errFromStatus := apistatus.ErrFromStatus(st)
+
+		if len(testItem.compatibleErrs) > 0 {
```

Comment:
Useless `if`?

Reply:
not all test cases have compatibleErrs array

Reply:
Then `for ... range` wouldn't do a thing irrespective of this `if`, isn't it?

Reply:
Yes, got you idea. Removed